test/signatures-caa.json

```diff
@@ -0,0 +1,84 @@
+[
+	{
+		"id": {
+			"$t": "https://spreadsheets.google.com/feeds/list/xyz123/1/public/values/1"
+		},
+		"updated": {
+			"$t": "2015-03-18T17:22:33.363Z"
+		},
+		"category": [
+			{
+				"scheme": "http://schemas.google.com/spreadsheets/2006",
+				"term": "http://schemas.google.com/spreadsheets/2006#list"
+			}
+		],
+		"title": {
+			"type": "text",
+			"$t": "7/11/2013 13:55:53"
+		},
+		"content": {
+			"type": "text",
+			"$t": "fullname: Scott González, emailaddress: scott.gonzalez@gmail.com, confirmation: I AGREE"
+		},
+		"link": [
+			{
+				"rel": "self",
+				"type": "application/atom+xml",
+				"href": "http://spreadsheets.google.com/feeds/list/xyz123/1/public/values/1"
+			}
+		],
+		"gsx$timestamp": {
+			"$t": "7/11/2013 13:55:53"
+		},
+		"gsx$fullname": {
+			"$t": "Scott González"
+		},
+		"gsx$emailaddress": {
+			"$t": "scott.gonzalez@gmail.com"
+		},
+		"gsx$confirmation": {
+			"$t": "I AGREE"
+		}
+	},
+	{
+		"id": {
+			"$t": "https://spreadsheets.google.com/feeds/list/xyz123/1/public/values/2"
+		},
+		"updated": {
+			"$t": "2015-03-18T17:22:33.363Z"
+		},
+		"category": [
+			{
+				"scheme": "http://schemas.google.com/spreadsheets/2006",
+				"term": "http://schemas.google.com/spreadsheets/2006#list"
+			}
+		],
+		"title": {
+			"type": "text",
+			"$t": "7/11/2013 15:36:17"
+		},
+		"content": {
+			"type": "text",
+			"$t": "fullname: Alexander Schmitz, emailaddress: arschmitz@gmail.com, confirmation: I AGREE"
+		},
+		"link": [
+			{
+				"rel": "self",
+				"type": "application/atom+xml",
+				"href": "http://spreadsheets.google.com/feeds/list/xyz123/1/public/values/2"
+			}
+		],
+		"gsx$timestamp": {
+			"$t": "7/11/2013 15:36:17"
+		},
+		"gsx$fullname": {
+			"$t": "Alexander Schmitz"
+		},
+		"gsx$emailaddress": {
+			"$t": "arschmitz@gmail.com"
+		},
+		"gsx$confirmation": {
+			"$t": "I AGREE"
+		}
+	}
```

Comment:
Might consider adding yourself the `ü` tends to cause issues in general if not properly handled. So would be good to have that in the tests.